### ISSUE TITLE
Autologin bug

### ISSUE THREAD

**Issue by jpixy** [butlerx/wetty#197]
Bug A:

I try to use autologin :
```
http://localhost:3000/wetty/ssh/term?sshpass=term
```
but it says:
```
term?sshpass=term@*****'s password: 
```
I run wetty with 
`sudo -u term wetty -p 3000 --sshhost localhost --sshport 22 --sshauth password`

Bug B:
```
➜  docker git:(johnny) ✗ sudo -u ibmadmin wetty -p 3000 --sshhost 9.42.25.106 --sshport 22 --sshauth password --sshuser ibmadmin --sshpass *****
{"message":"http on port 3000","level":"info"}
{"message":"Thu Jul 11 2019 04:40:41 GMT-0400 (EDT) Connection accepted.","level":"info"}
{"message":"PID=27754 STARTED on behalf of localhost","level":"info"}
{"message":"Exit with code: 127 PID=27754 ENDED","level":"info"}
{"message":"Thu Jul 11 2019 04:40:41 GMT-0400 (EDT) Connection accepted.","level":"info"}
{"message":"PID=27756 STARTED on behalf of localhost","level":"info"}
{"message":"Exit with code: 127 PID=27756 ENDED","level":"info"}
{"message":"9.200.96.59 - - [11/Jul/2019:08:40:48 +0000] \"GET /wetty/ssh HTTP/1.1\" 404 148 \"-\" \"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.100 Safari/537.36\"\n","level":"info"}
{"message":"9.200.96.59 - - [11/Jul/2019:08:40:52 +0000] \"GET /wetty HTTP/1.1\" 304 - \"-\" \"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.100 Safari/537.36\"\n","level":"info"}
{"message":"9.200.96.59 - - [11/Jul/2019:08:40:52 +0000] \"GET /wetty/public/index.js HTTP/1.1\" 304 - \"http://9.42.25.106:3000/wetty\" \"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.100 Safari/537.36\"\n","level":"info"}
{"message":"9.200.96.59 - - [11/Jul/2019:08:40:52 +0000] \"GET /wetty/public/index.css HTTP/1.1\" 304 - \"http://9.42.25.106:3000/wetty\" \"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.100 Safari/537.36\"\n","level":"info"}
{"message":"Thu Jul 11 2019 04:40:52 GMT-0400 (EDT) Connection accepted.","level":"info"}
{"message":"PID=27769 STARTED on behalf of localhost","level":"info"}
{"message":"Exit with code: 127 PID=27769 ENDED","level":"info"}
```
Than http://9.42.25.106:3000/wetty in browser,

It will alert "reconnect"

If I use `--sshauth publickey` it will be OK.

**Comment by butlerx** [comment]
could you please update to the latest version and confirm this is this an issue?

aswell could you post info on what os you are using

**Comment by butlerx** [comment]
This is now fixed in version 1.2
